app/frontend/src/components/forms/manage/AddTeamMember.vue

```diff
@@ -13,6 +13,11 @@
         return-object
         :search-input.sync="searchUsers"
       >
+        <!-- no data -->
+        <template #no-data>
+          <span>Can't find someone? They may not have joined the site.<br/>
+            Kindly send them a link to the site and ask them to log in.</span>
+        </template>
```

Comment:
Little janky without padding, could class the span and add style to this component for padding.
But does just using the v-autocomplete `no-data-text` attribute look better than the slot?

![image](https://user-images.githubusercontent.com/17445138/122461995-e2969580-cf68-11eb-9bce-557b7b1f1334.png)

Reply:
Added `px-2` padding to it.